module/os/linux/spl/spl-dcache.c

```diff
@@ -0,0 +1,48 @@
+/*
+ *  Copyright (C) 2019 OpenZFS Project
+ *  Written by Pavel Snajdr <snajpa@snajpa.net>
```

Comment:
Only a legal entity can formally hold copyright, which the OpenZFS project is not.
Also Copyright notices are not required anymore since quite a few years anyway.

If you want to add a copyright, you can add you own. 
Ownership of the code is with you, not with the project. You licence this code TO users of this project ;)